test/api/controllers/documentController.test.js

```diff
@@ -1285,6 +1285,56 @@ describe('DocumentController', () => {
       should(documentController.translateKoncorde)
         .be.calledWith({ equals: { name: 'Melis' } });
     });
+
+    it('should throw an error if one document has an error with the strict arg', async () => {
+      kuzzle.ask.withArgs('core:storage:public:document:updateByQuery').resolves({
+        successes: esResponse.successes,
+        errors: [
+          {
+            document: { _id: 'id3', _source: { foo: 'bar', bar: 'foo' } },
+            status: 206,
+            reason: 'reason'
+          }
+        ]
+      });
+
+      request.input.body = {
+        query: {
+          match: { foo: 'bar' }
+        },
+        changes: {
+          bar: 'foo'
+        }
+      };
+      request.input.args.refresh = 'wait_for';
+      request.input.args.source = true;
+      request.input.args.strict = true;
+      request.context.user = { _id: 'aschen' };
+
+      const methodName = 'core:storage:public:document:updateByQuery';
```

Comment:
Same thing here, if you use the `methodName` variable why not also use it on the `1290` line?

Reply:
Was for debugging purpose I think, useless now